tests/tracing/test_propagation.py

```diff
@@ -1,40 +0,0 @@
-import sentry_sdk
-import pytest
-
-
-def test_standalone_span_iter_headers(sentry_init):
```

Comment:
there is no longer a difference between a standalone span and a transaction so this is obsolete